.factorypath

```diff
@@ -1 +1,3 @@
-<factorypath />
\ No newline at end of file
+<factorypath>
```

Comment:
I'm not sure, what this file is for, but it contains a local file path, so I'm assuming it should be added to the gitignore file and be removed from the repo?